src/test/java/org/mdbenefits/app/testutils/Page.java

```diff
@@ -47,7 +47,7 @@ public String getHeader() {
 
     public void goBack() {
         await().atMost(20, TimeUnit.SECONDS).until(
-            () -> !(driver.findElements(By.id("back-link")).isEmpty())
+                () -> !(driver.findElements(By.id("back-link")).isEmpty())
```

Comment:
grumble, grumble, much of this file is just syntax formatting. I'll mark where there's an actual change.